README.md

```diff
@@ -163,6 +164,33 @@ let feedFromTwitter user pwd = taskSeq {
+| **Frameworks**             | .NET 5.0+, NetStandard 2.1                                                        | .NET 5.0+, NetStandard 2.0 and 2.1, .NET Framework 4.6.1+            |
+| **Underlying type**        | `System.Collections.Generic.IAsyncEnumerable<'T>`                                 | Its own type, also called `IAsyncEnumerable<'T>`, but not compatible |
+| **Implementation**         | State machine (statically compiled)                                               | No state machine, continuation style                                 |
+| **Semantics**              | `seq`-like: on-demand                                                             | `seq`-like: on-demand                                                |
+| **Support `let!`**         | All `task`-like: `Async<'T>`, `ValueTask<'T>`, `Task<'T>` or any `GetAwaiter()`   | `Async<'T>` only                                                     |
+| **Support `do!`**          | `Async<unit>`, `Task<unit>` and `Task`, `ValueTask<unit>` and `ValueTask`         | `Async<unit>` only                                                   |
+| **Support `yield!`**       | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq`                                                           |
+| **Support `for`**          | `IAsyncEnumerable<'T>`, `AsyncSeq`, any sequence                                  | `AsyncSeq` any sequence                                              |
+| **Behavior with `yield`**  | Zero allocations, no `Task` or even `ValueTask` created                           | Allocated, an F# `Async` wrapped in a singleton `AsyncSeq`           |
+| **Conversion to other**    | `TaskSeq.toAsyncSeq`                                                              | `AsyncSeq.toAsyncEnum`                                               |
+| **Conversion from other**  | Implicit (yield!) or `TaskSeq.ofAsyncSeq`                                         | `AsyncSeq.ofAsyncEnum`                                               |
+| **Recursion in `yield!`**  | No (requires F# support, upcoming)                                                | Yes                                                                  |
+| **Based on F# concept of** | `task`                                                                            | `async`                                                              |
+| **`MoveNextAsync`**        | `ValueTask<bool>`                                                                 | `Async<'T option>`                                                   |
+| **`Current` internals**    | `ValueOption<'T>`                                                                 | `Option<'T>`                                                         |
+| **Cancellation**           | Implicit token governing iteration                                                | Implicit token passed to each subtask                                |
```

Comment:
```suggestion
| **Cancellation**           | Implicit token governs iteration                                                | Implicit token flows to all subtasks per normal `Async` semantics                               |
```
Is "Implicit token governs iteration" -> "Implicit token governs iteration but requires explicit propagation of token to child Tasks"

Reply:
applied, tx